### PR TITLE
feat: Update Card to use new Image Component

### LINKED ISSUE #1687: Update Card to use new Image Component
Now that we have the  [Image component that allows gradual image loading](https://github.com/react-native-training/react-native-elements/pull/1582)  we should use it in the Card component.

### PR DESCRIPTION
**Pull request by haruelrovix**
Close #1687

### CODE REVIEW

File: src/card/Card.js [hunk 1]
```diff
@@ -80,9 +81,10 @@ const Card = props => {
 
         {image && (
           <View style={imageWrapperStyle && imageWrapperStyle}>
-            <BackgroundImage
+            <Image
               style={[{ width: null, height: 150 }, imageStyle && imageStyle]}
               source={image}
+              PlaceholderContent={<ActivityIndicator />}
```
Comment (iRoachie) [inline src/card/Card.js]: I'd say remove `PlaceholderContent` just to be consistent with the other components that use Image

**Comment by iRoachie** [comment]
I also updated your branch from next, so be aware of that when you start working locally again.

**Comment by iRoachie** [comment]
Also this will fail when it has children.

Simulator Screen Shot - iPhone 7 - 2019-08-01 at 08 59 46

**Comment by iRoachie** [comment]
#1992 Made the change so that the Image component uses ImageBackground so this problem should be solved now. Once you do a rebase we should be good to go

File: src/card/Card.js (hunk 1 above)
Comment (haruelrovix) [reply src/card/Card.js]: @iRoachie updated via [f8858bc](https://github.com/react-native-training/react-native-elements/pull/1835/commits/f8858bcd36ad9b2494c0f9a3e0e062e60009cb29).

**Comment by iRoachie** [comment]
Thanks again @haruelrovix real mvp 💯

**Pull request merged**